parser/package.json

```diff
@@ -20,7 +20,7 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#039f4c584655cb4b5e933c9058b84ead7c8c6aee",
```

Comment:
It looks like Expensify/App uses a different commit: https://github.com/Expensify/App/blob/06c7dac74ad5bcc8009c614570625afe4006f621/package.json#L103

Reply:
Yes, I'm now creating the PR with bumping version in Expensify/App as well so it will be aligned

Reply:
I'll wait to E/App PR to be merged, before this one